### PR TITLE
Centralize all config

### PR DESCRIPTION
**Pull request by harimohanraj89**
Story: https://app.shortcut.com/simpledotorg/story/7619/extract-all-environment-specific-configuration-to-a-single-place

Continuation of #3 to centralize the following configurations into a single place.

* [x] Instance ips
* [x] Inventory vars
* [ ] SSL cert names - punted for now
* [x] SSH keys
* [x] App env vars
* [x] Sidekiq concurrency vars
* [x] Monitoring secrets
* [x] Datadog secrets

### CODE REVIEW

File: group_vars/ethiopia_demo/vars.yml [hunk 1]
```diff
@@ -4,3 +4,24 @@ app_env: "demo"
 default_country: "ET"
 deploy_env: "ethiopia-demo"
 domain_name: "simple-demo.moh.gov.et"
+ssh_keys:
```
Comment (prabhanshuguptagit) [inline group_vars/ethiopia_demo/vars.yml:7]: ❤️

Comment (prabhanshuguptagit) [reply group_vars/ethiopia_demo/vars.yml:7]: Let's create a follow up story to replicate this to the mainline ansible?

File: group_vars/ethiopia_demo/vault.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
+$ANSIBLE_VAULT;1.1;AES256
```
Comment (prabhanshuguptagit) [inline group_vars/ethiopia_demo/vault.yml:1]: Let's alphabetise these while we're at it.

**Review by prabhanshuguptagit — APPROVED** [review]
Such clean much wow :shipit:

File: group_vars/ethiopia_demo/vault.yml (hunk 2 above)
Comment (harimohanraj89) [reply group_vars/ethiopia_demo/vault.yml:1]: Oops, lost sight of that halfway through, thanks.

File: group_vars/ethiopia_demo/vars.yml (hunk 1 above)
Comment (harimohanraj89) [reply group_vars/ethiopia_demo/vars.yml:7]: https://app.shortcut.com/simpledotorg/story/7732/simplify-ssh-key-setup

**Comment by harimohanraj89** [comment]
There are a couple tiny issues that need to be sorted out, but as this is relatively greenfield, I'm going to move quickly and merge this right away. Once we get the big picture architecture in place, we can refine the little details. In the meantime, the deployment repo is safely managing the actual deployment of our ET environments, so I'm not too worried.

**Pull request merged**